docs/reference/management_commands.md

```diff
@@ -152,6 +150,10 @@ Displays a summary of the contents of the references index. This shows the numbe
 
 Wagtail keeps a log of search queries that are popular on your website. On high traffic websites, this log may get big and you may want to clean out old search queries. This command cleans out all search query logs that are more than one week old (or a number of days configurable through the [`WAGTAILSEARCH_HITS_MAX_AGE`](wagtailsearch_hits_max_age) setting).
 
+```{versionchanged} 5.0
+The `Query` model has been moved to the [searchpromotions](editors_picks) module and this functionality can now be invoked with the [](searchpromotions_garbage_collect) command.
```

Comment:
I think it's better to mention the functionality itself rather than specifically mentioning the `Query` model here. Also, we should probably replace the description pointing to the new command entirely, with link to the release notes about the migration.